packages/hw-transport-node-hid/src/TransportNodeHid.js

```diff
@@ -229,7 +229,15 @@ export default class TransportNodeHid extends Transport<string> {
       return response;
     }
 
+    const { debug } = this;
     const deferred = defer();
+    if (debug) {
+      debug("=>" + apdu.toString("hex"));
```

Comment:
aaaaand I missed that missing space :scream:

Reply:
oops